src/App.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { FeedScreen } from '@src/App'
+import { render, fireEvent } from '@testing-library/react-native'
+import React from 'react'
+
+it('renders default elements', () => {
+  //const { } = render(<FeedScreen />)
```

Comment:
Did you add this? (I've not got this from my last pull of sprint 2 branch) Can this be removed if it's not doing anything?